pom.xml

```diff
@@ -38,17 +38,17 @@
     </licenses>
 
     <properties>
-        <kotlin.version>1.4.32</kotlin.version>
-        <dokka.version>1.4.20</dokka.version>
+        <kotlin.version>1.6.10</kotlin.version>
+        <dokka.version>1.6.10</dokka.version>
         <maven.compiler.source>8</maven.compiler.source>
```

Comment:
Should we bump this one to 11 as well? :)

Reply:
Yes why not actually!